ansible/roles/operator-pipeline/templates/openshift/tasks/github-pipelinerun-summary.yml

```diff
@@ -164,6 +164,13 @@ spec:
           fi
         fi
 
+
+        if [[ -f "$(workspaces.output.path)/merge_error.txt" ]]; then
+          MERGE_ERROR=$(cat "$(workspaces.output.path)/merge_error.txt")
+          echo -e "\n## ❌ Not merged\n" >> $PR_NAME/comment.md
+          echo -e "PR cannot be merged for the following reasons: \`$MERGE_ERROR\`" >> $PR_NAME/comment.md
```

Comment:
One of the message from the example you liked shows following message:

`PR cannot be merged for the following reasons: Pull request Pull request is closed (enablePullRequestAutoMerge)`

Does the double "Pull request" come directly from github?

Reply:
Yep, I've checked it out and it seems like there is multiple examples of error message where `Pull request` is duplicate. I can search for these cases and parse them, if we don't mind the difference between "official" error message.